pkg/controller/migmigration/dvm.go

```diff
@@ -76,6 +79,25 @@ func (t *Task) getDirectVolumeMigration() (*migapi.DirectVolumeMigration, error)
 	return nil, nil
 }
 
+func (t *Task) getDirectVolumeMigrationProgressForDVM(dvm *migapi.DirectVolumeMigration, destClient compat.Client) (*migapi.DirectVolumeMigrationProgress, error) {
+
+	dvmLabels := dvm.GetCorrelationLabels()
+	dvmLabels["directvolumemigration"] = string(dvm.UID)
```

Comment:
Same question about why we are modifying what comes out of `GetCorrelationLabels`

Reply:
Done, removed the redundant code.